modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommercePriceOnApplication.testcase

```diff
@@ -572,6 +639,362 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-11683. Buyer can use Price on Application with default setting"
+	@priority = 5
+	test CanUsePriceOnApplication {
```

Comment:
Can you change the name to CanUsePriceOnApplicationOnPriceList the one that was present in the ticket is wrong